### ISSUE TITLE
Comment field contents cannot be emptied when filled

### ISSUE THREAD

**Issue by alm-nl** [PowerDNS-Admin/PowerDNS-Admin#1516]
### PDA version

0.4.0

### Python version

3.6

### Steps to Reproduce

1. Create a comment (Edit, add comment and save)
2. Press "Save Changes" button (Apply Changes, close)
3. Remove the comment (Edit, remove comment and save)
4. Press "Save Changes" button (Apply Changes, close)
5. The removed comment will show up again

It is possible to change the comment, but it's not possible to clear it. Putting in some spaces does not work to clear it.

If the comment is removed of a similar named record (i.e. "@") of same type (I tested with TXT) and a new comment added to a similar named record (i.e. "@") with same type (again TXT) it will remove the comment from the first and add the new comment to the second record. If you change a different named (i.e. test) record with same type (TXT) it does not affect the comment on the "@" record. Removing the comment on the different named record (i.e. test) has the same result as with the "@" record of type TXT.

I'm using PowerDNS Auth 4.7.3 with MySQL backend.
PowerDNS-Admin is running in the docker container as supplied on https://hub.docker.com/r/powerdnsadmin/pda-legacy:latest
Previously I was running PowerDNS-Admin of 11 months ago (likely version 0.2.4) from https://hub.docker.com/r/ngoduykhanh/powerdns-admin
With old version this did not happen.
I've tested with different zones and they all show this behavior.
I've checked the open and closed issues for similar bug but could not find it.

If you have questions, please let me know.

### Expected Behavior

The expected behaviour would be that the comment can be removed and not show up again after "Save Changes" (Apply Changes and Close).

### Observed Behavior

Removing the comment will show it again after "Save Changes" (Apply Changes and Close) unless a new comment is added under the same named record with same type.

**Comment by AzorianMatt** [comment]
It seems this issue revolves around a scenario where there is a single record existing / remaining in an RRSet. I found I was able to remove comments from records when there was more than one matching record in the RRSet. Once I tried clearing the comment on the only remaining record in the RRSet that had a comment, the comment would return after saving.

**Comment by alm-nl** [comment]
Should I test with 0.4.1 (new latest as of this week)? If yes, I can hopefully do so tomorrow.
The quote in the comment you made before the latest, is that something from PowerDNS itself?

**Comment by AzorianMatt** [comment]
@alm-nl no, nothing new to test here. This is likely an issue in all versions based on the comment I found. That comment is in the current code base. From what I gather, it seems like a pretty nasty issue to try and overcome due to the general design of PDA Legacy.

This just goes on the list of reasons why PDA Next will become more of a synchronization strategy as opposed to the facade strategy that PDA Legacy currently creates on top of PowerDNS Authoritative server.

If you have noticed, I tagged this particular issue as `wont-fix`, because overcoming this is just not worth the effort. PDA Next will definitely solve this kind of problem much better.
